lib/vagrant/cli.rb

```diff
@@ -30,6 +31,20 @@ def execute
         return 0
       end
 
+      if @main_args.include?("-install-autocomplete")
+        # Install autocomplete scripts
+        written_paths = Vagrant::Util::InstallCLIAutocomplete.install
+        if written_paths.length > 0
+          @env.ui.info("Autocomplete installed at paths:")
+          written_paths.each do |p|
+            @env.ui.info("- #{p}")
+          end
+        else
+          @env.ui.info("Autocomplete not installed")
```

Comment:
We should use the `I18n` library here to abstract out the strings into the translations file.